### PR TITLE
added pushbutton

### PR DESCRIPTION
**Pull request by Abse2001**

### CODE REVIEW

File: src/source/any_source_component.ts [hunk 1]
```diff
@@ -21,6 +22,7 @@ export const any_source_component = z.union([
   source_simple_power_source,
   source_simple_battery,
   source_simple_inductor,
+  source_simple_pushbutton,
```
Comment (seveibar) [inline src/source/any_source_component.ts:25]: Please change to push_button

**Review by seveibar — CHANGES_REQUESTED** [review]
Can you change the filename and export name for push button

**Comment by Abse2001** [comment]
Ok

File: src/source/any_source_component.ts (hunk 1 above)
Comment (Abse2001) [reply src/source/any_source_component.ts:25]: Ok

Comment (seveibar) [reply src/source/any_source_component.ts:25]: Clarification: the full name should be simple_source_push_button

File: src/source/index.ts [hunk 2]
```diff
@@ -15,3 +14,4 @@ export * from "./source_group"
 export * from "./source_net"
 export * from "./source_simple_battery"
 export * from "./source_simple_inductor"
+export * from "./source_simple_pushbutton"
```
Comment (seveibar) [inline src/source/index.ts:17]: Fix is needed here

**Review by seveibar — APPROVED** [review]

**Pull request merged**